newdocs/docs/api/tracker-stores.mdx

```diff
@@ -3,7 +3,7 @@ id: tracker-stores
 sidebar_label: Tracker Stores
 title: Tracker Stores
 ---
-import variables from '@site/src/variables';
+import variables from '@site/docs/variables.json';
```

Comment:
why do we use the `@site` syntax here (while we explicitly don't for the others?)

Reply:
I omitted this one 😬